CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+### 3.6.29
+
+* Add `mips64` to support the platform of loongnix
+
```

Comment:
Please remove this. We prefer to fill CHANGELOG by ourselves.

Reply:
OK，I have pushed once.Thanks!

Reply:
OK,I have pushed once more.Thanks!